projects/1.20-fabric/assets/worldtools/worldtools/lang/zh_cn.json

```
@@ -0,0 +1,139 @@
+  "text.autoconfig.worldtools.option.world.worldGenerator.type.@Tooltip": "提示：VOID：生成一个空的世界。FLAT：生成一个超平坦世界。DEFAULT：生成一个默认世界。",
+  "text.autoconfig.worldtools.title": "WorldTools设置",
+  "worldtools.capture.and": "和",
+  "worldtools.capture.chunk": "%s个区块",
+  "worldtools.capture.chunks": "%s个区块",
+  "worldtools.capture.click_to_open": "%s（点击打开）",
+  "worldtools.capture.container": "%s个容器",
+  "worldtools.capture.containers": "%s个容器",
+  "worldtools.capture.entities": "%s个实体",
+  "worldtools.capture.entity": "%s个实体",
+  "worldtools.capture.in_dimension": "，位于维度",
+  "worldtools.capture.nothing_saved_yet": "还未保存%s的数据",
+  "worldtools.capture.player": "%s名玩家",
+  "worldtools.capture.players": "%s名玩家",
+  "worldtools.capture.saved": "已保存",
+  "worldtools.capture.saved.advancements": "已保存%s的进度",
+  "worldtools.capture.saved.chunks": "已保存区块，位于区块坐标%s和维度%s",
+  "worldtools.capture.saved.chunks.anonymized": "已保存区块，位于维度%s",
+  "worldtools.capture.saved.compressed": "将捕获的ZIP压缩包保存为%s",
+  "worldtools.capture.saved.end_flow": "完成%s的捕获",
+  "worldtools.capture.saved.entities": "已保存实体%s，位于区块%s和维度%s",
+  "worldtools.capture.saved.entities.anonymized": "已保存%s，位于维度%s",
+  "worldtools.capture.saved.levelData": "已保存%s属于%s",
+  "worldtools.capture.saved.mapData": "已保存维度%s的地图",
+  "worldtools.capture.saved.metadata": "已保存%s的元数据",
+  "worldtools.capture.saved.player": "已保存玩家%s，位于坐标%s和维度%s",
+  "worldtools.capture.saved.player.anonymized": "已保存玩家%s，位于维度%s",
+  "worldtools.capture.saved.statistics": "已保存%s的数据",
+  "worldtools.capture.to_directory": "到保存目录",
+  "worldtools.capture.took": "（耗时%s）",
+  "worldtools.gui.browser.title": "WorldTools浏览器",
+  "worldtools.gui.capture.existing_world_confirm.message": "一个名叫\"%s\"的存档已经存在，是否覆盖？（将与现有数据合并）",
+  "worldtools.gui.capture.existing_world_confirm.title": "发现重复存档",
+  "worldtools.gui.escape.button.finish_download": "保存%s的捕获数据",
+  "worldtools.gui.manager.button.cancel": "取消",
+  "worldtools.gui.manager.button.config": "设置",
+  "worldtools.gui.manager.button.start_download": "开始下载",
+  "worldtools.gui.manager.button.stop_download": "停止下载",
+  "worldtools.gui.manager.title": "WorldTools管理器",
+  "worldtools.gui.manager.world_name_placeholder": "输入世界名称...（默认：%s）",
+  "worldtools.key.categories": "WorldTools",
+  "worldtools.key.open_config": "打开配置",
+  "worldtools.key.toggle_capture": "快速下载",
+  "worldtools.log.error.already_capturing": "%s的捕获任务已经在运行中！",
+  "worldtools.log.error.failed_to_create_session": "无法为\"%s\"创建会话！（错误：%s）",
+  "worldtools.log.error.failed_to_save_level": "无法为\"%s\"保存level.dat！（错误：%s）",
+  "worldtools.log.error.failed_to_visit_file": "无法访问文件%s以创建ZIP压缩包！（错误：%s）",
+  "worldtools.log.error.failed_to_zip": "无法创建会话以压缩捕获数据%s！（错误：%s）",
+  "worldtools.log.error.not_capturing": "没有捕获任务在运行！",
+  "worldtools.log.error.world_name_too_long": "世界名称\"%s\"过长！（最大%d个字符）",
+  "worldtools.log.info.singleplayer_capture": "实验性功能警告：单人游戏中的捕获是实验性功能，可能不会按预期工作！",
+  "worldtools.log.info.started_capture": "开始捕获%s...",

```

```suggestion
  "worldtools.log.info.started_capture": "开始捕获%s……",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。